ts/modules/draggable-points.src.ts

```diff
@@ -2962,6 +2962,11 @@ function mouseDown(
         return;
     }
 
+    // Ignore if dragging an annotation
+    if ((chart as any).hasDraggedAnnotation) {
```

Comment:
Please do not cast to `any`. Instead go in this file up to the namespace and add the property as an optional boolean to the chart interface (and add a mini doclet, that it requires the module).

Reply:
Done, thanks! :)